assets/components/src/button-card/style.scss

```diff
@@ -0,0 +1,104 @@
+/**
+ * Button Card
+ */
+
+@import '../../../shared/scss/colors';
+@import '~@wordpress/base-styles/colors';
+
+.newspack-button-card {
+	border-radius: 3px;
+	border: 1px solid $light-gray-500;
+	box-shadow: inset 0 0 0 2px transparent;
+	cursor: pointer;
+	display: block;
+	margin: 32px 0;
+	padding: 32px;
+	text-decoration: none;
+	transition: background-color 125ms ease-in-out, border-color 125ms ease-in-out,
+		box-shadow 125ms ease-in-out;
+
+	&:active,
+	&:focus,
+	&:hover {
+		background-color: $light-gray-100;
+		color: $primary-600;
+		text-decoration: none;
+
+		.title,
+		.desc {
+			color: inherit;
+		}
+	}
+
+	&:focus {
+		border-color: $primary-500;
+		box-shadow: inset 0 0 0 2px $primary-500 !important;
+		outline: none !important;
+		position: relative;
+	}
+
+	.title,
+	.desc {
+		line-height: 24px;
+		transition: color 125ms ease-in-out;
+	}
+
+	.title {
+		color: $dark-gray-900;
+		font-size: 17px;
+		font-weight: bold;
+	}
+
+	.desc {
+		color: $dark-gray-300;
+		font-size: 14px;
+	}
+
+	// Grouped
+
+	&.grouped {
+		border-radius: 0;
+
+		& + & {
+			margin-top: -33px;
+		}
+
+		&.br--bottom {
+			border-bottom-left-radius: 3px;
+			border-bottom-right-radius: 3px;
+		}
+
+		&.br--top {
+			border-top-left-radius: 3px;
+			border-top-right-radius: 3px;
+		}
+	}
+
+	// Icon
+
+	&.has-icon {
+		align-items: center;
+		display: grid;
+		grid-gap: 16px;
+		grid-template-columns: 48px auto;
+
+		&:active,
+		&:focus,
+		&:hover {
+			svg {
+				fill: currentColor;
+			}
+		}
+
+		svg {
+			fill: $primary-500;
+			transition: fill 125ms ease-in-out;
+		}
+	}
+
+	// Is Small
+
+	&.is-small {
```

Comment:
The `is-small` styles don't look much different from the regular styles. Wonder if we should make the type styles smaller as we do with the `is-small` version of the `ActionCard` component? For example, the card from Campaigns:

<img width="1041" alt="Screen Shot 2021-05-14 at 1 52 38 PM" src="https://user-images.githubusercontent.com/2230142/118322086-cad87700-b4bb-11eb-9b25-18e433775106.png">

vs. the new `ButtonCard` with small styles:

<img width="915" alt="Screen Shot 2021-05-14 at 1 52 50 PM" src="https://user-images.githubusercontent.com/2230142/118322111-d5930c00-b4bb-11eb-93b7-ddd917b7ede2.png">

Reply:
We're not using isSmall so I kind of created a very basic style but you're right, it should look different from the regular size...  Anyway, I pushed some changes in https://github.com/Automattic/newspack-plugin/pull/961/commits/97d9c099313fbce70d700d145c869611f5fb2e72

I also added a `chevron` option... Why not?! 😂

Reply:
Nice, looks much more consistent now!